src/validators/common.ts

```diff
@@ -214,6 +217,14 @@ export type IdentifierSection = Record<string, IdentifierT & {
 	type: number;
 }>;
 
+export type AnnotationSection = {
+	content?: string;
```

Comment:
Should this also have `lastRevisionId: number | null;` ?

Reply:
Good question. Currently all these `*Section` types only have the properties which are being validated and are exclusively used by [the import consumer](https://github.com/kellnerd/bookbrainz-utils/blob/421cbc9e9783ca5e957b6291efa378e2a3519bd8/importer/src/consumer/validators/index.ts#L208) (for validation).
In that case `lastRevisionId` is not needed, but we might want to use the types for other use cases in the future, for example for the JSON payload which is sent to the entity editor handler.
But then all the other types here would have to be reworked as well to add more "optional" properties (they may not be optional in the TS sense, but in the sense of validation).

Reply:
Yes we're thinking along the same lines.
I don't  think we should be worrying about this now.